pkg/webhook/resources/setting/validator.go

```diff
@@ -1060,3 +1062,24 @@ func validateKubeConfigTTLSetting(newSetting *v1beta1.Setting) error {
 func validateUpdateKubeConfigTTLSetting(_ *v1beta1.Setting, newSetting *v1beta1.Setting) error {
 	return validateKubeConfigTTLSetting(newSetting)
 }
+
+func validateDefaultSnapshotMaxCount(setting *v1beta1.Setting) error {
+	if setting.Value == "" {
+		return nil
+	}
+
+	num, err := strconv.ParseInt(setting.Value, 10, 64)
+	if err != nil {
+		return werror.NewInvalidError(err.Error(), "value")
+	}
+
+	if num < 2 || num > 250 {
+		return werror.NewInvalidError("snapshot max count should be between 2 and 250", "value")
```

Comment:
Should this actually be the value of `settings.DefaultSnapshotMaxCount.GetInt()`, rather than hard-coding 250?